# Student Marks Analysis

## Objective
To analyze student marks using Python and calculate the total, average, highest, and lowest marks.

In [1]:
# Student marks
marks = [78, 85, 92, 67, 74, 88, 95, 81, 69, 90]

marks

[78, 85, 92, 67, 74, 88, 95, 81, 69, 90]

In [2]:
total = sum(marks)

total

819

In [3]:
average = total / len(marks)

average

81.9

In [4]:
highest = max(marks)

highest

95

In [5]:
lowest = min(marks)

lowest

67

In [6]:
print("Student Marks Analysis")
print("----------------------")
print("Total Marks   :", total)
print("Average Marks :", average)
print("Highest Mark  :", highest)
print("Lowest Mark   :", lowest)

Student Marks Analysis
----------------------
Total Marks   : 819
Average Marks : 81.9
Highest Mark  : 95
Lowest Mark   : 67


## Conclusion

The student marks were successfully analyzed using Python.

- Total marks: **819**
- Average marks: **81.9**
- Highest mark: **95**
- Lowest mark: **67**

This analysis demonstrates how Python lists and built-in functions such as `sum()`, `max()`, `min()`, and `len()` can be used to perform basic student data analysis.